package.json

```diff
@@ -1,15 +1,56 @@
 {
   "name": "birthdayy-discord-bot",
   "version": "0.2.3",
-  "main": "dist/index.js",
+  "description": "Birthday Discord Bot built on Sapphire Framework",
   "author": "@chillihero",
+  "license": "Apache-2.0",
+  "main": "./dist/index.js",
+  "imports": {
```

Comment:
what are these imports for?

Reply:
Not to have to go a relative path that would be really too long